### PR TITLE
Add automation: "URLSSLVerification

### PR DESCRIPTION
**Pull request by JonathanMeler**
Output sample:
image

### CODE REVIEW

**Comment by JonathanMeler** [comment]
@liorkol please review the script's string as well (description, results, input/output description, etc..)

File: Scripts/script-URLSSLVerification.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      "EntryContext": ec
+  })
+type: python
+tags:
+- url
+- Enrichment
+comment: Verify URL SSL certificate
+system: true
+args:
+- name: url
```
Comment (gshriki) [inline Scripts/script-URLSSLVerification.yml]: should be set as array no?

Comment (JonathanMeler) [reply Scripts/script-URLSSLVerification.yml]: Fixed

**Comment by JonathanMeler** [comment]
@liorkol ?

**Comment by JonathanMeler** [comment]
"URL Enrichment - Generic" changes:
url_enrichment_-_generic_copy_wed_feb_28_2018

**Pull request merged**